internal/runners/clean/run_lin_mac.go

```diff
@@ -127,5 +127,15 @@ func removeInstall(cfg configurable) error {
 		aggErr = errs.Wrap(aggErr, "Failed to remove system files at %s: %v", appPath, err)
 	}
 
+	installPath, err := installation.InstallPath()
+	if err != nil {
+		aggErr = errs.Wrap(aggErr, "Could not get installation path")
+	}
+
+	err = os.RemoveAll(installPath)
```

Comment:
Can we make this only remove all if the path is empty? Otherwise pring a warning (don't error).

I'm really scared otherwise of how this would combine with people installing to custom dirs, or legacy installations that installed to eg. `/usr/local/bin`. I'd like to stay away from removing dirs recursively as much as possible.

Reply:
Good point!